contracts/DXswapFeeSetter.sol

```diff
@@ -0,0 +1,44 @@
+pragma solidity =0.5.16;
+
+import './interfaces/IDXswapFactory.sol';
+
+contract DXswapFeeSetter {
+    address public owner;
+    mapping(address => address) public pairOwners;
+    IDXswapFactory public factory;
+  
+    constructor(address _owner, address _factory) public {
+        owner = _owner;
+        factory = IDXswapFactory(_factory);
+    }
+
+    function transferOwnership(address newOwner) external {
+        require(msg.sender == owner, 'DXswapFeeSetter: FORBIDDEN');
```

Comment:
Maybe use a custom modifier to clean the code a bit here?

Reply:
I thought on it, but just for the sake of making the contract very short and minimal I didnt added it, I think it is pretty clear that if the sender of the call is not the owner it will fail.
This was approved by @jpkcambridge and @nicoelzer in a previous PR, so I think we are good, if not I would have to re deploy everything again.
I check this condition reverted in all functions in the tests.